src/commands/config/create.ts

```diff
@@ -1,13 +1,12 @@
 import fs from "fs";
 import { error, info, success } from "signale";
 import { Argv } from "yargs";
-import { createTempDNS } from "../../implementations/config/create";
 import { deployDocumentStore } from "../../implementations/deploy/document-store/document-store";
 import { deployTokenRegistry } from "../../implementations/deploy/token-registry/token-registry";
+import { handler as CreateTempDns } from "../dns/txt-record/create";
```

Comment:
- bad case
- dont use shorter name (Temp => Temporary)

Reply:
i have changed this and some of the variables that are using shorter naming as well.